### PR TITLE
just the web side of things

### PR DESCRIPTION
**Pull request by braswelljr**

### CODE REVIEW

File: web/next.config.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ const withPWA = require('next-pwa')({
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  output: 'export',
```
Comment (profclems) [inline web/next.config.js:13]: Let's set the output directory to the `dist` folder using `distDir: 'dist',`
```suggestion
  output: 'export',
  distDir: 'dist',
```

Comment (profclems) [reply web/next.config.js:13]: I'm going to go ahead to apply this suggestion

**Comment by braswelljr** [comment]
Don't upgrade next yet there is an issue with the version `13.4.8` open telemetry versions are conflicting between next `13.4.9` and contentlayer there are issues on that

File: web/src/app/page.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      </header>
+      {/* main */}
+      <div className="px-5 sm:px-8 lg:px-16">
+        {/* form */}
+        <form action="#" method="post" onSubmit={handleSubmit(onSubmit)} className="">
+          <div className="mx-auto max-w-lg space-y-4 px-2">
+            {/* label */}
+            <div className={cn('flex flex-col gap-5')}>
+              <label htmlFor="command-input" className="font-bold">
+                Enter Command to Generate `<em>docker-compose.yml</em>` *
```
Comment (profclems) [inline web/src/app/page.tsx:81]: Let's change this to "Paste your `docker run` command below!"

The "docker run" should be a link pointing to the reference page https://docs.docker.com/engine/reference/run/

File: web/src/app/page.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  return (
+    <main className={cn('')}>
+      {/* header */}
+      <header className="flex min-h-[40vh] flex-col justify-center px-5 sm:min-h-[35vh] sm:px-8 lg:px-16">
+        <div className="">
+          <div className="mx-auto max-w-lg space-y-8">
+            <h1 ref={titleRef} className="text-center text-4xl font-bold uppercase">
+              {siteConfig.name}
+            </h1>
+            <p className="text-center text-lg">{siteConfig.description}</p>
```
Comment (profclems) [inline web/src/app/page.tsx:69]: Let's change the description to something more sellable. Maybe something along the lines of:

> Say goodbye to complex 'docker run' commands and embrace the power of `$ docker-compose up`! Compozify generates a clean and concise docker-compose.yml file for you from your `docker run` commands.

**Review by profclems — CHANGES_REQUESTED** [review]

File: web/src/app/page.tsx (hunk 3 above)
Comment (braswelljr) [reply web/src/app/page.tsx:69]: Alright.

**Review by profclems — APPROVED** [review]

**Pull request merged**